test/fw/ptl/lib/ptl_server.py

```diff
@@ -4289,12 +4301,8 @@ def update_attributes(self, obj_type, bs, overwrite=False):
                     else:
                         self.nodes[id].attributes.update(binfo)
                 else:
-                    if 'Mom' in binfo:
-                        self.nodes[id] = get_mom_obj(self, binfo['Mom'], binfo,
-                                                     snapmap={NODE: None})
-                    else:
-                        self.nodes[id] = get_mom_obj(self, id, binfo,
-                                                     snapmap={NODE: None})
+                    self.nodes[id] = MoM(self, id, binfo,
```

Comment:
Hey @monkeystate , 
Are these changes to ptl_server.py intentional ? 
The original changes were done recently as part https://github.com/openpbs/openpbs/pull/2094

Reply:
I think I messed it up when the ptl_server.py was created and I moved over the two functions into it.  It should be correct now.  I synced it with master again and pulled the two functions over.